packages/solana-nft-adapter/.gitignore

```diff
@@ -0,0 +1,106 @@
+node_modules/
```

Comment:
having two gitignores in the repo can be confusing, do you mind merging the two together on the root?

Reply:
removed